Codeunits/Codeunit_50000.txt

```diff
@@ -0,0 +1,105 @@
+OBJECT Codeunit 50000 Encryption
+{
+  OBJECT-PROPERTIES
+  {
+    Date=06.09.17;
+    Time=09:14:13;
+    Modified=Yes;
+    Version List=NAVCOMMUNITY10.00;
+  }
+  PROPERTIES
+  {
+    OnRun=BEGIN
+          END;
+
+  }
+  CODE
+  {
+
+    PROCEDURE GenerateMD5Hash@50000(SourceString@50007 : Text) : Text;
+    VAR
+      MD5@50000 : DotNet "'System.Core, Version=3.5.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'.System.Security.Cryptography.MD5Cng";
+      hashBytes@50002 : DotNet "'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'.System.Array";
+      byte@50003 : DotNet "'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'.System.Byte";
+      stringBuilder@50004 : DotNet "'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'.System.Text.StringBuilder";
+      Encoding@50005 : DotNet "'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'.System.Text.Encoding";
+      i@50006 : Integer;
+    BEGIN
+      IF SourceString = '' THEN
+        EXIT;
+
+      //--- Using BEGIN
```

Comment:
Please do not use comments  -unless they are necessary to explain what goes on.

Reply:
This is exactly what these comments do, because they explain why I dispose and therefore mark the security area.